solutions/scripts/analytics/lib-analytics.js

```diff
@@ -0,0 +1,409 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/**
+ * Customer's XDM schema namespace
+ * @type {string}
+ */
+const CUSTOM_SCHEMA_NAMESPACE = '_sitesinternal';
```

Comment:
This should be the namespace for bitdefender

Reply:
@iuliag - we change it to `_bitdefender`. It is ok?